coalahtml/_coalahtml/app/styles/files.css

```diff
@@ -1,7 +1,20 @@
+.highlight-line {
+    background: #9E5660 !important;
```

Comment:
Not sure but we need it. I had to add important to override the inherited colors.  :sweat_smile: